consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/ErrorControllerAdvice.scala

```diff
@@ -31,11 +32,16 @@ class ErrorControllerAdvice {
   ))
   def handle_404 = new ResponseEntity(NOT_FOUND)
 
-  @ExceptionHandler
-  def handle_500(e: Throwable) = new ResponseEntity(ErrorCode(e), INTERNAL_SERVER_ERROR)
+  @ExceptionHandler(Array(
+    classOf[MethodArgumentTypeMismatchException]
+  ))
+  def handle_400(e: MethodArgumentTypeMismatchException) = new ResponseEntity(ErrorMsg(e.getMessage), BAD_REQUEST)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method name does not match the regular expression '^([a-z][A-Za-z0-9]*(?:_=)?|<<|>>>|>|==|!=|<|<=|>|>=|\||&|\^|\+|-|\*|\/|%|\|\||&&|\+\+|--|\+=|-=)$'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4491035)